pkg/api/resource/quantity.go

```diff
@@ -105,6 +105,7 @@ const (
 	DecimalExponent = Format("DecimalExponent") // e.g., 12e6
 	BinarySI        = Format("BinarySI")        // e.g., 12Mi (12 * 2^20)
 	DecimalSI       = Format("DecimalSI")       // e.g., 12M  (12 * 10^6)
+	Digit           = Format("Digit")           // e.g., 1, 2, 3...
 )
```

Comment:
It doesn't need to, but with Digit, I can write 1,2,3 to the yaml directly for nvidia GPU, with DecimalSI, I should write 1M,2M,3M or modify the unit inside kubelet/kube-scheduler.